test/io.jl

```diff
@@ -760,7 +760,7 @@ end
     df = DataFrame(
         A=Int64[1,4,9,16,25,36,49,64],
         B = [
-            md"[DataFrames.jl](http://juliadata.github.io/DataFrames.jl)",
+            md"ABC",
```

Comment:
handling of links in MD changed in nightly. But since it is unable I propose just not to test it.